src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -204,7 +204,7 @@ bool AletheProofPostprocessCallback::update(Node res,
     // VP8: (cl (=> (and F1 ... Fn))) (cl (not (=> (and F1 ... Fn) false))
```

Comment:
This should be VP8: (cl (not (=> (and F1 ... Fn) false))

Reply:
Actually I think it's `// VP8: (cl (=> (and F1 ... Fn) false))`?

Reply:
Ah yes of course!